thrall/app/lib/elasticsearch/ElasticSearch.scala

```diff
@@ -53,8 +54,11 @@ class ElasticSearch(config: ElasticSearchConfig, metrics: Option[ThrallMetrics])
     List(response.map(_ => ElasticSearchBulkUpdateResponse()))
   }
 
-  def indexImage(id: String, image: Image, lastModified: DateTime)
+  def indexImage(id: String, image: Image, lastModified: DateTime, reindex: Boolean = false)
                 (implicit ex: ExecutionContext, logMarker: LogMarker): List[Future[ElasticSearchUpdateResponse]] = {
+
+    val index = if(reindex){accretorAlias}else{imagesAlias}
```

Comment:
appreciate the word reindex is becoming overloaded (as per PR description) but would calling it `reIndexAlias` be better?

Reply:
or `reIngestAlias` (given other reindex -> reingest name changes in this PR)

Reply:
We've gone for `migration` as the word.

This PR is mostly for design discussion now as the change I was attempting had too many preconditions.
https://github.com/guardian/grid/pull/3324